### PR TITLE
Update some DG diagrams to make them more readable

### PR DESCRIPTION
**Pull request by oeggy03**
Also fixed a small error which used to state that the delete c 1 command deletes a person in the Logic section.

### CODE REVIEW

**Comment by oeggy03** [comment]
For issue #110

File: docs/images/CompanyModelDiagram.png
Comment (Nauxe) [inline docs/images/CompanyModelDiagram.png:1]: Good eye! Omitting extra details about an internship improves readability

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ The sections below give more details of each component.
 
 The **API** of this component is specified in [`Ui.java`](https://github.com/AY2324S1-CS2103T-T10-4/tp/tree/master/src/main/java/seedu/address/ui/Ui.java)
 
-![Structure of the UI Component](images/UiClassDiagram.png)
+<img src="images/UiClassDiagram.png" width="1200" />
```
Comment (Nauxe) [inline docs/DeveloperGuide.md:73]: I'm confused about these changes, the diagram sizes look fine to me in the current DG.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -96,13 +96,13 @@ taking `execute("delete p 1")` API call as an example.
...
 
-![Interactions Inside the Logic Component for the `delete p 1` Command](images/DeletePersonSequenceDiagram.png)
+<img src="images/DeletePersonSequenceDiagram.png" width="1200" />
 
 When executing a command that affects Company objects, the logic works similarly.
 
 The sequence diagram below takes `execute("delete c 1")` API call as an example. 
-The command `delete c 1` deletes the first person listed in the list of people in the addressbook.
-![Interactions Inside the Logic Component for the `delete p 1` Command](images/DeleteCompanySequenceDiagram.png)
+The command `delete c 1` deletes the first company listed in the list of companies in the addressbook.
```
Comment (Nauxe) [inline docs/DeveloperGuide.md:104]: Impressively observant in noticing this small error!

**Review by Nauxe — APPROVED** [review]
LGTM! Just confused about the part where you used HTML tags in `docs/DeveloperGuide.md`.

**Comment by Tim-Siu** [comment]
Thank you for picking up typos

**Pull request merged**